src/main/java/com/myecommerce/MyECommerce/controller/MemberController.java

```diff
@@ -57,24 +55,18 @@ public ResponseEntity<MemberDto> signUpCustomer(@Valid @RequestBody MemberDto me
      * 회원 로그인 post /member/signin
      **/
     @PostMapping("/signin")
-    public ResponseEntity<String> signIn(@RequestBody MemberDto memberDto) {
-        // 1. 아이디, 패스워드 일치여부 확인
-        MemberDto authenticatedMember = memberService.authenticateMember(memberDto);
-
-        // 2. JWT 토큰 생성
-        String token = jwtAuthenticationProvider.createToken(authenticatedMember);
-
-        // 3. 토큰 반환
-        return ResponseEntity.ok(token);
+    public ResponseEntity<String> signIn(@RequestBody RequestMemberDto memberDto) {
+        // 사용자검증 후 JWT 토큰 반환
+        return ResponseEntity.ok(memberService.signIn(memberDto));
     }
 
     /**
      * 회원 로그아웃 post /member/signout
      **/
     @PostMapping("/signout")
-    public ResponseEntity<String> signOut(@RequestHeader Map<String, Object> headers) {
-        // 로그아웃
-        memberService.signOut(headers.get("authorization").toString());
+    public ResponseEntity<String> signOut(@RequestHeader @Name("Authorization") String authorization) {
```

Comment:
```suggestion
    public ResponseEntity<String> signOut(@RequestHeader(HttpHeaders.AUTHORIZATION) String authorization) {
```
위와 같은 방식으로 제안드립니다~! HttpHeaders 는 Spring core 에 있는 상수를 사용하시면 될 듯 합니다.